.vscode/settings.json

```diff
@@ -6,8 +6,11 @@
         }
     ],
     "eslint.options": {
-        "rulePaths": ["../scripts/eslint/built/rules/"],
+        "rulePaths": ["./scripts/eslint/built/rules/"],
```

Comment:
I'm surprised changing this allowed it to continue to work. For me, if the only change I make is the below, everything works fine.

Reply:
All the other paths are relative to the workspace root, though, so it kind of makes sense?

Reply:
When I do this locally, it breaks my setup :( it all depends on where the working directory of the eslint process is. Apparently on my machine, it's in the `src` folder, and it's not clear to me why. Maybe we should just set these to true and keep the leading `..` so it's unambiguous?

Reply:
Sorry, "set these to true" means `changeProcessCWD: true` in both cases.

Reply:
Same here. If I copy updated file - no error when I open `tests/cases/fourslash/addMemberInDeclarationFile.ts`, but fails in `src/services/findAllReferences.ts`
```
Error: ENOENT: no such file or directory, scandir 'c:\web\projects\TypeScript\src\scripts\eslint\built\rules'
```

If I change only `"eslint.workingDirectories"` it's reversed with error when `tests/cases/fourslash/addMemberInDeclarationFile.ts` is opened.